src/Symfony/Component/VarDumper/Tests/Fixtures/NotLoadableClass.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Symfony\Component\VarDumper\Tests\Fixtures;
+
+class NotLoadableClass extends NotLoadableClass
```

Comment:
a class extending itself ? Are you missing the time of your Patchwork framework ?